### ISSUE TITLE
Wav2Vec2: Input Normalization Clarification

### ISSUE THREAD

**Issue by apoorv2904** [pytorch/fairseq#2691]
## Is the raw wave input to the wav2vec2 model zero mean and unit variance normalized?

The [Wav2Vec2.0 paper](https://arxiv.org/pdf/2006.11477.pdf) mentions that "The raw waveform input to
the encoder is normalized to zero mean and unit variance."

However, in the finetuning examples as well as the ckpt args have normalize set to False. Could you please clarify if the input is supposed to normalized or not?

Thank you for help on this.

Apoorv

**Comment by mcggood** [comment]
I also got errors saying no arg "normalize" in dict. and I simple comment out the _assert lines_
